observer/observer_config.py

```diff
@@ -0,0 +1,13 @@
+# -*- coding: utf-8 -*-
+
+OBSERVER_ID = 'SATNOGS-1'
```

Comment:
Let's move the configuration to ``settings.py`` keeping the same format (default value that we override with env vars).